pkg/middleware/promhttp_middleware_test.go

```diff
@@ -100,7 +100,7 @@ func assertMetricExists(t *testing.T, data []byte, name string, labels map[strin
 			return
 		}
 	}
-	assert.Fail(t, "unable to find metric '%s' with labels '%v'", name, labels)
+	assert.Fail(t, "unable to find metric with labels", "name", name, "labels", labels)
```

Comment:
the last arg to the `assert.*` functions is called `msgAndArgs` and is actually printf-formatted (see https://github.com/stretchr/testify/blob/master/assert/assertions.go#L316)

```suggestion
	assert.Fail(t, "unable to find metric with labels", "name: %s, labels: %v", name, labels)
```

Reply:
ohhh, that's not the same logic as the controller logger! 🤦‍♂️